src/chrome/content/rules/GPGTools.org.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Different content http/https:
+		support.gpgtools.org
```

Comment:
Uhm, no, you're right. For some reason, I thought https://support.gpgtools.org didn't redirect properly. I will update it.